src/utils/request.js

```diff
@@ -42,30 +56,20 @@ service.interceptors.response.use(
    * Here is just an example
    * You can also judge the status by HTTP Status Code
    */
-  response => {
+  async response => {
     const res = response.data
 
-    // if the custom code is not 20000, it is judged as an error.
-    if (res.code !== 20000) {
+    // 50008: Illegal token; 50012: Other clients logged in; 50014: Token expired;
+    if (res.code === 50008 || res.code === 50012 || res.code === 50014) {
```

Comment:
maybe the code would be more understandable and shorter as follows:
```javascript
if ([50008, 50012, 50014].includes(res.code))
```

Maybe you could change it if you like my proposal.

Reply:
yes, you are right